### PR TITLE
Remove Startup.cs

### PR DESCRIPTION
**Pull request by Toxantron**
I updated the structure to match ASP.NET Core 6 and above. I also updated the port back to 5000.

### CODE REVIEW

File: src/MyApplication.App/Program.cs [hunk 1]
```diff
@@ -14,26 +20,95 @@ public static void Main(string[] args)
         {
```
Comment (1nf0rmagician) [inline src/MyApplication.App/Program.cs]: At least for all .net tutorials from Microsoft they use Top-Level-Statements now. Do we want to go one step further and use those for the template as well? 🤔

Comment (Toxantron) [reply src/MyApplication.App/Program.cs]: Sure, let's go all the way.

File: src/MyApplication.App/Properties/launchSettings.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "https://localhost:60572;http://localhost:60573"
+      "applicationUrl": "http://localhost:5000"
```
Comment (1nf0rmagician) [inline src/MyApplication.App/Properties/launchSettings.json]: Please make the default https 😊

Comment (Toxantron) [reply src/MyApplication.App/Properties/launchSettings.json]: Of course, good catch

**Pull request merged**